packages/client-core/src/systems/ui/MediaControlsUI.tsx

```diff
@@ -45,21 +59,29 @@ type MediaControlsProps = {
 const MediaControlsView = (props: MediaControlsProps) => {
   const mediaComponent = useHookstate(getMutableComponent(props.entity, MediaComponent))
 
+  const controlsPercent = '10%'
```

Comment:
i think these belong in a common file, as proper constants

Reply:
replaced these variables with constant values direct in the styles